src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/GoogleAnalytics4DataSourceItemFixture.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Reveal.Sdk.Dom.Core.Extensions;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class GoogleAnalytics4DataSourceItemFixture
+    {
+        [Fact]
+        public void GoogleAnalytics4DataSourceItem_IsDataSourceItem_WhenConstructed()
```

Comment:
I do not think we need this test case. 
This will hard-code GoogleAnalytics4DataSourceItem to be a DataSourceItem's type, which can be changed.
By unit tests for the parent class that GoogleAnalytics4DataSourceItem inherits from, we can ensure that the inherited behaviors are working correctly

Reply:
my idea about why this test case is important is because user may want to use data sources in an abstracted way. For example, you can have:
```
List<DataSource> dts = [];
dts.add(GoogleAnalytic4Datasource());
dts.add(ExcelDatasource());
...
```
Or just the ability to change the data source dynamically.

For that to happen the class must inherit the DataSource/DataSourceItem correctly. IMO It's like an important feature of the library.

Reply:
I don't feel strongly about this, but I don't think this test makes a lot of sense. This isn't really testing anything useful. We want to test functionality, not the object inheritance. Plus, if it wasn't a DataSourceItem, then none of the other tests would pass because the properties depend on the base class to function properly.

Reply:
Let's remove these types of tests. That don't really add any value. The test essentially checks that the inheritance hierarchy is correct, which is something that should be clear from reading the code. Tests are typically more valuable when they verify dynamic behavior or ensure correct functioning rather than checking static relationships.